src/EnergyPlus/GeneratorFuelSupply.cc

```diff
@@ -93,40 +91,26 @@ namespace GeneratorFuelSupply {
     // REFERENCES:
     // Annex 42 documentation
 
-    // Using/Aliasing
-    using namespace DataGenerators;
-
     void GetGeneratorFuelSupplyInput(EnergyPlusData &state)
     {
 
         // SUBROUTINE INFORMATION:
         //       AUTHOR         B Griffith
         //       DATE WRITTEN   July 2006,
-        //       MODIFIED       na
         //       RE-ENGINEERED  this module extracted from older SOFC module for
         //                      reuse with both Annex 42 models,
 
-        // Using/Aliasing
-        using Curve::GetCurveIndex;
-        using DataLoopNode::ObjectIsNotParent;
-        using NodeInputManager::GetOnlySingleNode;
-        using ScheduleManager::GetScheduleIndex;
-
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         //  INTEGER                     :: GeneratorNum !Generator counter
-        int NumAlphas;                 // Number of elements in the alpha array
-        int NumNums;                   // Number of elements in the numeric array
-        int IOStat;                    // IO Status when calling get input subroutine
         Array1D_string AlphArray(25);  // character string data
         Array1D<Real64> NumArray(200); // numeric data TODO deal with allocatable for extensible
-        bool ErrorsFound(false);       // error flag
-        int FuelSupNum;
-        std::string ObjMSGName;
-        int ConstitNum;
-        auto &cCurrentModuleObject = state.dataIPShortCut->cCurrentModuleObject;
```

Comment:
[src/EnergyPlus/GeneratorFuelSupply.cc:117]:(style),[variableScope],The scope of the variable 'NumAlphas' can be reduced.
[src/EnergyPlus/GeneratorFuelSupply.cc:118]:(style),[variableScope],The scope of the variable 'NumNums' can be reduced.
[src/EnergyPlus/GeneratorFuelSupply.cc:119]:(style),[variableScope],The scope of the variable 'IOStat' can be reduced.
[src/EnergyPlus/GeneratorFuelSupply.cc:122]:(style),[variableScope],The scope of the variable 'ErrorsFound' can be reduced.
[src/EnergyPlus/GeneratorFuelSupply.cc:123]:(style),[variableScope],The scope of the variable 'FuelSupNum' can be reduced.
[src/EnergyPlus/GeneratorFuelSupply.cc:124]:(style),[variableScope],The scope of the variable 'ObjMSGName' can be reduced.
[src/EnergyPlus/GeneratorFuelSupply.cc:125]:(style),[variableScope],The scope of the variable 'ConstitNum' can be reduced.
[src/EnergyPlus/GeneratorFuelSupply.cc:126]:(style),[variableScope],The scope of the variable 'cCurrentModuleObject' can be reduced.